ch2driver/pytpcc/drivers/abstractdriver.py

```diff
@@ -156,18 +158,74 @@ def getOneCH2PPDoc(self, tableName, fieldValues, generateKey):
             val[columns[l]] = v1
         return key, val
 
-    def genNestedDoc(self, fieldValues, tableName):
+    def getOneCH2PPFlatDoc(self, tableName, fieldValues, generateKey):
```

Comment:
In CH2+ we only have one phone and address per customer - do we need some logic in here to do the same thing?

Reply:
We will not have a CH2PF variant.